contrib/ansible/roles/master/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+    - { role: kubernetes }
```

Comment:
Yeah you are right. In this case since kubernetes role depends on common, common is now an implicit dependency fro master. This should have common in the list to be explicit.